boskos/ranch/storage.go

```diff
@@ -114,10 +114,11 @@ func (s *Storage) DeleteResource(name string) error {
 
 // UpdateResource updates a resource if it exists, errors otherwise
 func (s *Storage) UpdateResource(resource *crds.ResourceObject) (*crds.ResourceObject, error) {
+	resource.Namespace = s.namespace
```

Comment:
Well we need something that enforces the namespace. Another approach could be to add a wrapper around the client itself, but I found it more logical in the storage, because it must be configured with a namespace anyways, as it has some `Get(name string)` funcs